packages/client/src/interface/channels/text/Composition.tsx

```diff
@@ -186,11 +186,110 @@ export function MessageComposition(props: Props) {
   function onKeyDownMessageBox(
     event: KeyboardEvent & { currentTarget: HTMLTextAreaElement }
   ) {
+    const insideCodeBlock = isInCodeBlock(event.currentTarget.selectionStart);
+    const usingBracketIndent = (event.ctrlKey && (event.key === "[" || event.key === "]"));
```

Comment:
This should be `⌘ + [` and `⌘ + ]` on macOS, as seen further in the code this is currently dealt with by doing:
```js
(event.ctrlKey || event.metaKey)
```
This is fine for this PR as this will be solved at some point in #185.

Reply:
should be fixed now with [a6742e7](https://github.com/revoltchat/frontend/pull/182/commits/a6742e7351cdcf3a143ee8dc27a21c4abb9b320b)